ts/components/session/SessionInboxView.tsx

```diff
@@ -53,13 +58,17 @@ export class SessionInboxView extends React.Component<any, State> {
       return <></>;
     }
 
+    const persistor = persistStore(this.store);
```

Comment:
this will  store the whole redux store right? Can you make it store only the settings slice?

Reply:
Yup you can. There's a whitelist property in createStore added in: da00ac8 . I set it to only include user config slice.